src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravVurderingQueries.kt

```diff
@@ -44,20 +45,38 @@ fun Connection.createAktivitetskravVurdering(
     }
 }
 
+const val queryUpdateAktivitetskravVurdering =
+    """
+        UPDATE AKTIVITETSKRAV_VURDERING SET status=?, stoppunkt_at=?, updated_at=?, beskrivelse=?, updated_by=? WHERE uuid = ?
+    """
+
+fun Connection.updateAktivitetskravVurdering(
+    aktivitetskravVurdering: AktivitetskravVurdering,
+) {
+    this.prepareStatement(queryUpdateAktivitetskravVurdering).use { preparedStatement ->
+        preparedStatement.setString(1, aktivitetskravVurdering.status.name)
+        preparedStatement.setDate(2, Date.valueOf(aktivitetskravVurdering.stoppunktAt))
+        preparedStatement.setObject(3, nowUTC())
+        preparedStatement.setString(4, aktivitetskravVurdering.beskrivelse)
+        preparedStatement.setString(5, aktivitetskravVurdering.updatedBy)
+        preparedStatement.setString(6, aktivitetskravVurdering.uuid.toString())
+        preparedStatement.execute()
+    }
```

Comment:
Bra å legge inn en sjekk på at nøyaktig en rad ble oppdatert (men må kanskje bruke executeUpdate for å få det ut).

Reply:
Ja, kan legge til det.